src/App.test.js

```diff
@@ -16,9 +16,17 @@ it("renders without crashing", () => {
   shallow(<App />);
 });
 
```

Comment:
Why is there no spec that authInfo is valid only if expires_time is within range?